README.md

```diff
@@ -104,64 +209,218 @@ await memphis.close()
 ```
 
 ### Creating a Station
-**Unexist stations will be created automatically through the SDK on the first producer/consumer connection with default values.**<br><br>
-_If a station already exists nothing happens, the new configuration will not be applied_
-
-```python
-station = memphis.station(
-  name="<station-name>",
-  schema_name="<schema-name>", # defaults to "" (no schema)
-  retention_type=Retention.MAX_MESSAGE_AGE_SECONDS, # MAX_MESSAGE_AGE_SECONDS/MESSAGES/BYTES/ACK_BASED(cloud only). Defaults to MAX_MESSAGE_AGE_SECONDS
-  retention_value=604800, # defaults to 604800
-  storage_type=Storage.DISK, # Storage.DISK/Storage.MEMORY. Defaults to DISK
-  replicas=1, # defaults to 1
-  idempotency_window_ms=120000, # defaults to 2 minutes
-  send_poison_msg_to_dls=True, # defaults to true
-  send_schema_failed_msg_to_dls=True, # defaults to true
-  tiered_storage_enabled=False, # defaults to false
-  partitions_number=1, # defaults to 1 
-  dls_station="<station-name>" # defaults to "" (no DLS station) - If selected DLS events will be sent to selected station as well
-)
+**A station will be automatically created for the user when a consumer or producer is used if no stations with the given station name exist.**<br><br>
```

Comment:
@John-Memphis , please add a phrase on what a station is

Reply:
👍